fennel/datasets/datasets.py

```diff
@@ -1203,7 +1205,7 @@ def lookup(
                 res[col] = res[col].apply(
                     lambda x: parse_json(type_annotation, x)
                 )
-            return res.replace({np.nan: None}), found
+            return res, found
```

Comment:
very cool. So now the backend itself returns pd.NA ?

Reply:
Yup, or we patch that afterwards.